.env.production.local.example

```
@@ -0,0 +1,134 @@
+
+# ============================================
+# PostgreSQL Configuration
+# ============================================
+# PostgreSQL 16 + pgvector による構造化データとベクトル検索
+
+POSTGRES_HOST=localhost
+POSTGRES_PORT=5432
+POSTGRES_DB=context_store
+POSTGRES_USER=context_store_user
+# 強固なパスワードに変更してください（例: openssl rand -base64 32）
+POSTGRES_PASSWORD=CHANGE_ME_TO_STRONG_PASSWORD
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSTGRES_PASSWORD key should go before the POSTGRES_PORT key

(UnorderedKey)